tajo-catalog/tajo-catalog-server/src/main/resources/schemas/postgresql/postgresql.xml

```diff
@@ -151,7 +151,6 @@ xsi:schemaLocation="http://tajo.apache.org/catalogstore ../DBMSSchemaDefinition.
   				PID INT NOT NULL PRIMARY KEY,
   				TID INT NOT NULL,
   				PARTITION_NAME VARCHAR(128),
-  				ORDINAL_POSITION INT NOT NULL,
```

Comment:
This schema is not compatible with that of other databases.
It looks that some changes are missed.